src/test/groovy/com/autonomousapps/model/declaration/VariantSpec.groovy

```diff
@@ -6,25 +6,28 @@ class VariantSpec extends Specification {
 
   def "can compute variant from configuration (#configurationName => #variant)"() {
     expect:
-    Variant.of(configurationName) == variant
+    Variant.of(configurationName, [
+      'main', 'release', 'debug', 'test', 'testDebug', 'testRelease',
+      'flavor', 'flavorRelease', 'flavorDebug', 'testFlavorRelease', 'testFlavorDebug'
+    ] as Set<String>) == variant
 
     where:
     configurationName                  | variant
     'implementation'                   | new Variant('main', SourceSetKind.MAIN)
     'debugImplementation'              | new Variant('debug', SourceSetKind.MAIN)
-    'releaseFlavorImplementation'      | new Variant('releaseFlavor', SourceSetKind.MAIN)
+    'flavorReleaseImplementation'      | new Variant('flavorRelease', SourceSetKind.MAIN)
```

Comment:
I checked with the [Android docs](https://developer.android.com/studio/build/dependencies#dependency_configurations) and with an example project. The naming scheme is always:
`$sourceSet$flavor$buildType$scope`

(sourceSet is empty in case of 'main')

IMO some of the test data here was wrong (switching 'flavor' and 'buildType' order). These won't work anymore as they do not exist in a real Android project.

Reply:
Seems like I had a copy-paste error.